src/components/directory-content.js

```diff
@@ -122,6 +125,8 @@ const DirectoryContent = () => {
     const [mousePosition, setMousePosition] =
         React.useState(initialMousePosition);
 
+    const [dialogToOpen, setDialogToOpen] = useState(null);
```

Comment:
Rather [dialogOpen, setDialogOpen] ?

Reply:
The state contains the name of the dialog that we want to open, that's why I have chosen "dialogToOpen"